raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -2,10 +2,9 @@ version: 2
 
 settings:
   gas_price: "fast"
-  chain: goerli
```

Comment:
We don't need to do this now, but I think it would be nice to have the option of specifying multiple chain URLS in a dictionary in the environment file and pick one here.

Reply:
Why would you want to pick a chain in the scenario, as opposed to in the environment?

Reply:
I guess for the CI use-case it doesn't make too much sense, but for local testing it could be nice.

Also not important for now.